source/confirm_references.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Confirm references
+weight: 50
+---
+
+# Confirm references submitted
+The references of the candidate have been submitted.
```

Comment:
See comment about consistency for `confirm_conditions`.

Reply:
done, not it is called `submit references` (from the cukes)